packages/frontend-core/src/components/grid/cells/AttachmentCell.svelte

```diff
@@ -106,7 +106,7 @@
         on:change={e => onChange(e.detail)}
         maximum={maximum || schema.constraints?.length?.maximum}
         {processFiles}
-        {handleFileTooLarge}
+        handleFileTooLarge={isCloud ? handleFileTooLarge : null}
```

Comment:
This will just need to be `$props.isCloud` after updating the import from context above.